libs/community/langchain_community/chat_models/gigachat.py

```diff
@@ -133,6 +133,7 @@ def _build_payload(self, messages: List[BaseMessage], **kwargs: Any) -> gm.Chat:
         )
 
         payload.functions = kwargs.get("functions", None)
+        payload.model = self.name
```

Comment:
should this be `payload.model = self.model`?

Reply:
Yes, you're right. `self.model` is the correct thing. I got confused by the constructor pulling the `name` kwarg from the deepest parent class. Fixed.